docs/integrations/optuna.rst

```diff
@@ -43,22 +45,33 @@ Monitor the metrics and parameters checked at each run:
 .. code-block:: python3
 
     study = optuna.create_study(direction='maximize')
-    study.optimize(objective, n_trials=100, callbacks=[monitor])
+    study.optimize(objective, n_trials=100, callbacks=[neptune_callback])
 
 Log all results
 ---------------
+You can log additional information from optuna study after the sweep has completed.
+By running:
+
+.. code-block:: python3
 
-The script logs the following to Neptune:
+    opt_utils.log_study_info(study)
+
+You log the following things to Neptune:
 
 * Best score
 * Best parameters
-* plot_convergence figure
-* plot_evaluations figure
-* plot_objective figure
+* Interactive plotly figures from optuna.visualization: plot_contour, plot_slice, plot_parallel_coordinate, optimization_history
```

Comment:
These guys are not showed in the docs as image, while they give nice value.

Gif is overview, not a specific feature presentation.